### ISSUE TITLE
BLE001 (blind-except) add option to flag even if logger.exception / exc_info is used

### ISSUE THREAD

**Issue by BaxHugh** [astral-sh/ruff#23705]
### Summary

I would like there to be an option to disable this behaviour:

As documented at https://docs.astral.sh/ruff/rules/blind-except/
"Exceptions that are logged with exc_info enabled will not be flagged, as this is a common pattern for propagating exception"
```python
try:
    foo()
except BaseException:
    logging.exception("Something went wrong")
```

i.e. even if logging.exception is written, if the exception isn't re-raised then the lint rule should still flag, if this proposed option is set.

The issues with unexpected behaviour will still occur in this case, it's just that there will be an error log. In my codebases, if an exception is being suppressed, it needs to be a specific exception to avoid issues.

**Comment by dev-miro26** [comment]
Hi, @ntBre 
Can I work on this issue?
If allow, please assign to me.
Thank you.

**Comment by ntBre** [comment]
This issue is labeled needs-decision and isn't ready for implementation.

**Comment by dev-miro26** [comment]
Thank you for your quick reply.

**Comment by MichaReiser** [comment]
@dev-miro26 please read our contribution guidelines (https://github.com/astral-sh/ruff/blob/main/CONTRIBUTING.md)

**Comment by codingdave** [comment]
Hi, I second the feature request. My team has these rules

- always log exceptions with logger.exception() and then react to it (we always want to see the stacktrace)
- only at entrypoint catching exceptions of type Exception are allowed. They should be more specific everywhere else.

ruff can not help enforce these rules.
